### PR TITLE
Frontend: Add analytics, minor polish

### PR DESCRIPTION
**Pull request by Evalir**
Adds Analytics for the frontend client, and a very very minor code / visual polish.

### CODE REVIEW

File: website/src/pages/profile.js [hunk 1]
```diff
@@ -294,11 +297,6 @@ function DaoProfile ({ daoAddress, history }) {
   }
 
   const { organisation } = data
-  // if the organisation itself is null, it means
-  // it was not found in the DB
-  if (!organisation) {
-    return <ProfileNotFound history={history} />
-  }
```
Comment (onbjerg) [inline website/src/pages/profile.js:301]: How do empty profiles work now?

Comment (Evalir) [reply website/src/pages/profile.js:301]: This was actually redundant; this case is handled further up: https://github.com/1Hive/apiary/blob/master/website/src/pages/profile.js#L80

File: website/src/index.html [hunk 2]
```diff
@@ -1,26 +1,59 @@
...
+    <!-- Twitter -->
+    <meta name="twitter:creator" content="@1hiveorg" />
+    <!-- Misc !-->
+  </head>
 
-	<body>
-		<noscript>Your browser does not support JavaScript. Please update to a modern browser to use Apiary.</noscript>
+  <body>
+    <noscript
+      >Your browser does not support JavaScript. Please update to a modern
```
Comment (onbjerg) [inline website/src/index.html:27]: Typo

File: website/src/index.html [hunk 3]
```diff
@@ -1,26 +1,59 @@
...
+        cly.type = "text/javascript";
+        cly.async = true;
+        cly.src = "https://analytics.aragon.org/sdk/web/countly.min.js";
+        cly.onload = function() {
+          Countly.init();
+        };
+        var s = document.getElementsByTagName("script")[0];
+        s.parentNode.insertBefore(cly, s);
+      })();
+      console.log('atibo')
```
Comment (onbjerg) [inline website/src/index.html:56]: Remove

File: website/src/index.html [hunk 4]
```diff
@@ -1,26 +1,59 @@
...
-		<div id="app"></div>
+    <div id="app"></div>
 
-		<script src="index.js"></script>
-	</body>
+    <script src="index.js"></script>
+    <script type="text/javascript">
+      var Countly = Countly || {};
+      Countly.q = Countly.q || [];
+      //provide countly initialization parameters
```
Comment (onbjerg) [inline website/src/index.html:37]: No reason for the comment (nit)

File: website/src/index.html (hunk 2 above)
Comment (Evalir) [reply website/src/index.html]: Done in https://github.com/1Hive/apiary/pull/188/commits/df0d82f150aef65bc514fbdf9d65d765a0051a76

File: website/src/index.html (hunk 3 above)
Comment (Evalir) [reply website/src/index.html]: Done in https://github.com/1Hive/apiary/pull/188/commits/df0d82f150aef65bc514fbdf9d65d765a0051a76

**Comment by onbjerg** [comment]
Thanks ✨

File: website/src/index.html (hunk 4 above)
Comment (Evalir) [reply website/src/index.html]: Done in https://github.com/1Hive/apiary/pull/188/commits/df0d82f150aef65bc514fbdf9d65d765a0051a76

**Pull request merged**